packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -97,24 +96,7 @@ class BuildApkCommand extends BuildSubCommand {
       targetArchs: stringsArg('target-platform').map<AndroidArch>(getAndroidArchForName),
     );
     validateBuild(androidBuildInfo);
-
```

Comment:
\cc @xster re: removing app bundle wording

Reply:
I'm generally in favor of keeping this text if possible. Though we can reword for brevity if needed or offload some of the messages to a website link.

Reply:
I could see how this is useful for developers new to flutter, but for everyone that has run this command more than once they're not going to read it again. build apk is also one of the most used build commands, and the current wall of text makes it very difficult to tell what the null safety mode is

Reply:
> for everyone that has run this command more than once they're not going to read it again

You could say that about null safety. Once it's strong it's just noise.

Reply:
PS I'm in favor of this change or one that greatly reduces the wall of text about app bundles.

Reply:
We're currently less than 1% strong mode, so yes, once we hit a certain point it will be retired too

Reply:
Let's make this text shorter. It could even be a one line to https://flutter.dev/docs/deployment/android#when-should-i-build-app-bundles-versus-apks. 

Null safety migrations can be checked in and just done once. This "mistake" can be made each time someone releases.

Reply:
why don't we put that in the command documentation instead.

Reply:
Agree it should be in the help as well. I suspect most people don't read the help but I don't feel that strongly about it either since we also document it in https://flutter.dev/docs/deployment/android#building-the-app-for-release.